ddl/ddl_test.go

```diff
@@ -56,6 +56,9 @@ func (d *ddl) SetInterceptor(i Interceptor) {
 // JobNeedGCForTest is only used for test.
 var JobNeedGCForTest = jobNeedGC
 
+// NewSession is only used for test.
+var NewSession = newSession
```

Comment:
I don't think it is a good idea. I write a demo to show ```internal``` method in https://github.com/pingcap/tidb/pull/39698. If we use internal. we can use it in every ddl submodule. so we have more possibility to make ddl modular.

Reply:
It is fine here because the purpose is to avoid the other packages to use `newSession`. About making it modular, we can do this in another PR like https://github.com/pingcap/tidb/pull/39698.

Reply:
👌